ecl/resource_ecl_network_load_balancer_v2.go

```diff
@@ -0,0 +1,1384 @@
+		if err != nil {
+			if _, ok := err.(eclcloud.ErrDefault404); ok {
+				log.Printf("[DEBUG] Successfully deleted ECL Load Balancer Syslog Server %s",
+					loadBalancerSyslogServerID)
+				return n, "DELETED", nil
+			}
+			return n, "PROCESSING", err
+		}
+
+		log.Printf("[DEBUG] ECL Load Balancer Syslog Server %s still active.\n", loadBalancerSyslogServerID)
+		return n, "PROCESSING", nil
+	}
+}
+
+func createLoadBalancerSyslogServer(syslogConfig map[string]interface{}, loadBalancerID string, networkClient *eclcloud.ServiceClient, d *schema.ResourceData) error {
+	createSyslogOpts := load_balancer_syslog_servers.CreateOpts{
+		AclLogging:                  syslogConfig["acl_logging"].(string),
+		AppflowLogging:              syslogConfig["appflow_logging"].(string),
+		DateFormat:                  syslogConfig["date_format"].(string),
+		Description:                 syslogConfig["description"].(string),
+		IPAddress:                   syslogConfig["ip_address"].(string),
+		LoadBalancerID:              loadBalancerID,
+		LogFacility:                 syslogConfig["log_facility"].(string),
+		LogLevel:                    syslogConfig["log_level"].(string),
+		Name:                        syslogConfig["name"].(string),
+		PortNumber:                  syslogConfig["port_number"].(int),
+		TcpLogging:                  syslogConfig["tcp_logging"].(string),
+		TenantID:                    syslogConfig["tenant_id"].(string),
+		TimeZone:                    syslogConfig["time_zone"].(string),
+		TransportType:               syslogConfig["transport_type"].(string),
+		UserConfigurableLogMessages: syslogConfig["user_configurable_log_messages"].(string),
+	}
+
+	if elem, ok := syslogConfig["priority"]; ok {
+		i := elem.(int)
+		createSyslogOpts.Priority = &i
+	}
+
+	log.Printf("[DEBUG] Create Load Balancer Syslog Server Options: %#v", createSyslogOpts)
+	syslogServer, err := load_balancer_syslog_servers.Create(networkClient, createSyslogOpts).Extract()
+	if err != nil {
+		return fmt.Errorf("error creating ECL load balancer syslog server: %s", err)
+	}
+
+	log.Printf("[INFO] Load Balancer Syslog Server ID: %s", syslogServer.ID)
+	log.Printf("[DEBUG] Waiting for Load Balancer Syslog Server (%s) to become ACTIVE", syslogServer.ID)
+
+	stateConf := &resource.StateChangeConf{
+		Pending:      []string{"PENDING_CREATE"},
+		Target:       []string{"ACTIVE"},
+		Refresh:      waitForLoadBalancerSyslogServerComplete(networkClient, syslogServer.ID),
+		Timeout:      d.Timeout(schema.TimeoutCreate),
+		Delay:        5 * time.Second,
+		PollInterval: loadBalancerCreatePollInterval,
+		MinTimeout:   10 * time.Second,
+	}
+
+	_, err = stateConf.WaitForState()
+	if err != nil {
+		return fmt.Errorf(
+			"error waiting for load balancer syslog server (%s) to become ACTIVE: %s",
+			syslogServer.ID, err)
+	}
+	return nil
+}
+
+func flattenLoadBalancerInterfaces(in []load_balancer_interfaces.LoadBalancerInterface) []map[string]interface{} {
```

Comment:
schemaの型に沿って以下のように書いたほうが良いと思います。
```suggestion
func flattenLoadBalancerInterfaces(in []load_balancer_interfaces.LoadBalancerInterface) []interface{} {
```

Reply:
Suggestionの通り修正しました。